distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc

```diff
@@ -0,0 +1,63 @@
+:title: High Availability Initial Setup Exceptions
+:type: subInstalling
+:status: published
+:parent: High Availability Initial Setup
+:project: ${branding}
+:order: 00
+
+==== {title}
+
+These steps are handled differently for the initial setup of a Highly Available Cluster.
+
+===== HTTP Proxy Integration
+
+In order to integrate with an HTTP proxy, the ${branding} node's system properties must be changed to publish the correct port to external systems and users.
+This must be done before installing the first ${branding} node.
+There are two internal port properties that must be changed to whatever ports the ${branding} will run on on its system.
+Then there are two external port properties that must be changed to whatever ports the HTTP proxy is forwarding traffic through.
+
+[WARNING]
+====
+Make sure that the HTTP proxy is already running and forwarding traffic on the chosen ports before starting the ${branding}.
+There may be unexpected behavior otherwise.
+====
+
+In the below example, the HTTP proxy is forwarding https traffic via 8993 and http traffic via 8181.
+The ${branding} node will run on 1111 for https and 2222 for http (on the virtual machine that it's hosted on).
+[source]
+----
+org.codice.ddf.system.httpsPort=8993
+org.codice.ddf.system.httpPort=8181
+org.codice.ddf.system.port=${org.codice.ddf.system.httpsPort}
+org.codice.ddf.system.internalHttpsPort=1111
+org.codice.ddf.system.internalHttpPort=2222
+----
+
+===== Identical Directory Structures
+
+The two ${branding} nodes need to be in identical directories on their corresponding systems.
```

Comment:
❗️ We need not just the directory structure to be identical but also its location (same drive for Windows) from the root directory.

Reply:
Should we say something like "Have the same <DDF_HOME> directory" or be under the same root directory?

Reply:
❓  Should we list a warning of what the consequence is for not following this requirement? (cannot use migration to copy configurations)